packages/IMAPClient-UI.package/ICFolderDialog.class/class/addContactIcon.st

```diff
@@ -1,5 +1,5 @@
 constants
-addUserIcon
+addContactIcon
```

Comment:
when is this icon used?

As far as I understood, we won't need the plusContact icon in ICFolderdialog

Reply:
That's right, I will move the icon as soon as the other pull request is merged (that's why this pull request was marked as a draft ;)

Reply:
Ok, nice